askui_example/logging/logger.ts

```diff
@@ -1,10 +1,28 @@
 import "@askui/jest-allure-circus";
+import * as os from 'os';
 
 declare const allure: {
     step(name: string, body?: () => void): void;
     attachment(name: string, content: string, type: string): void;
+    writeEnvironmentInfo(info: Record<string, string>): void;
 }
 
+// Generate environment info
+const deviceId = `${os.hostname()}-${Math.random().toString(36).substring(2, 8).toUpperCase()}`;
+const workspaceId = process.env.ASKUI_WORKSPACE_ID || 'default-workspace-id';
+
+// Define environment info by creating function in allure. 
+allure.writeEnvironmentInfo({
+    APP_URL: `https://app.askui.com/workspaces/${workspaceId}/quick-start`,
+    Device_ID: deviceId,
+    Workspace_ID: workspaceId,
+    TEST_RUNNER: 'Jest',
+    PLATFORM: os.platform(),
+    OS_VERSION: os.release(),
+    NODE_VERSION: process.version,
+    TIMESTAMP: new Date().toISOString()
```

Comment:
Can we use here an uniform writing? e.g. UPPER_SNAKE_CASE
```suggestion
    APP_URL: `https://app.askui.com/workspaces/${workspaceId}/quick-start`,
    DEVICE_ID: deviceId,
    WORKSPACE_ID: workspaceId,
    TEST_RUNNER: 'Jest',
    PLATFORM: os.platform(),
    OS_VERSION: os.release(),
    NODE_VERSION: process.version,
    TIMESTAMP: new Date().toISOString()
```